local/src/core/ProxyVerifier.cc

```diff
@@ -275,12 +279,27 @@ Session::write(HttpHeader const &hdr)
   return zret;
 }
 
+swoc::Rv<int>
+Session::poll(std::chrono::milliseconds timeout)
+{
+  swoc::Rv<int> zret{-1};
```

Comment:
I fixed `Errata` to be better about this. You don't need to construct an `Rv` instance, you can instead do
```
return ::poll(...);
```
and expect it to work. For the error case you could do
```
return { 0, Errata().diag("...") };
```

Reply:
Oh, that is nice.